### PR TITLE
V1 of Spartakus

### PR DESCRIPTION
**Pull request by thockin**
Starting with some sample code I hacked and slashed.  This seems to work.  Reviewing will be painful, I suggest just perusing the codebase, ignoring godeps.

### CODE REVIEW

**Comment by thockin** [comment]
@sarahnovotny @aronchick

File: Makefile [hunk 1]
```diff
@@ -15,7 +15,7 @@
 .PHONY: all push container clean
 
 BIN = spartakus
-PKG = k8s.io/spartakus
```
Comment (philips) [inline Makefile]: aside: any chance we could get k8s.io/incubator or something?

**Review by philips — APPROVED** [review]
LGTM

**Comment by thockin** [comment]
regarding import paths.  We have, thus far, reserved k8s.io/foo as an alias
to github.com/kubernetes/foo.  I could do something special for incubator,
but this is currently a manual process, so I'd rather not.

**Pull request merged**